### DISCUSSION TITLE
Video of Diffbot with tracks

### DISCUSSION THREAD

**Discussion by Russ76** [fjp/2wd-robot#75] [Show and tell]
https://youtu.be/RnCINLTZ2cc

Thanks for all your hard work with Diffbot!

**Comment by fjp** [comment]
Awesome 😎 thank you Russ for sharing your success with Diffbot. Glad to see it is of use 😊 Roaming around and the mapping process look very good!

Thanks also to you for the contributions and patiently reporting issues to test and thereby improve the software. This really helps the project and will benefit others.
